scripts.js

```diff
@@ -0,0 +1,17 @@
+$(function(){
+	var carouselList = $("#carousel ul");
+	setInterval(changeSlide, 3000); 
+
+	function changeSlide(){
+	carouselList.animate({'marginLeft':-400}, 500, moveFirstSlide);
+	}
+	changeSlide();
+
+	function moveFirstSlide(){
+		var firstItem = carouselList.find("li:first");
+		var lastItem = carouselList.find("li:last");
+		lastItem.after(firstItem);
+		carouselList.css({marginLeft:0});
+	}
+	moveFirstSlide();
```

Comment:
you doesn't have to call this function - it will work because it is called in line 6